### DISCUSSION TITLE
Cart not working in incognito/private mode

### DISCUSSION THREAD

**Discussion by Stawik0001** [PrestaShop/PrestaShop#34999] [Q&A] [answered]
Hello everyone,

I think I have a rather unusual problem - adding products to the cart from a certain category (even two) does not work in incognito/private mode. It simply takes you to the cart with the information - "There are no products in the cart yet". In normal browser mode, everything works fine. In debug mode, when trying to add to cart, the following error occurs (attachment). Maybe someone has an idea for solving this product?
<img width="1728" alt="Zrzut ekranu 2024-01-8 o 20 58 37" src="https://github.com/PrestaShop/PrestaShop/assets/90532614/dc2f4f68-1244-47cc-93c9-5fbc8a0be06c">

**Comment by MatShir** [comment, marked as answer]
It seems there is a bug in the pscarbanner module. This module appears to have been unmaintained for some time now. I recommend deactivating it and finding a more appropriate module.

**Reply by MatShir** [reply]
You can try to right a report [here](https://github.com/PrestaShopCorp/pscartbanner) to notify the developer

**Comment by Stawik0001** [comment]
I turned off the module, but another error appeared.
<img width="1024" alt="Zrzut ekranu 2024-01-11 o 09 20 09" src="https://github.com/PrestaShop/PrestaShop/assets/90532614/d541cc62-24d4-43dc-b6ea-a3c6de2a250e">

**Reply by MatShir** [reply]
If you are still blocked, I advise you to check for the paid support or try other community platform https://www.prestashop-project.org/support/
